src/event/metric.rs

```diff
@@ -257,11 +257,23 @@ impl Metric {
         self
     }
 
-    /// Create a new Metric from this with all the data but marked as absolute.
-    pub fn to_absolute(&self) -> Self {
+    pub fn into_parts(self) -> (MetricSeries, MetricData) {
```

Comment:
Why is this method needed when the caller can do

```rust
let Metric { series, data } = metric;
```

since the fields are (currently?) public. A simple text-search on this PR also doesn't show any usage of the `into_parts` method.

Reply:
I think I added that early on with the expectation it would be useful, but obviously never found a use. I'll drop it.